src/array/ops/image.rs

```diff
@@ -58,17 +58,7 @@ impl<'a> DaftImageBuffer<'a> {
         with_method_on_image_buffer!(self, width)
     }
 
-    pub fn channels(&self) -> u16 {
```

Comment:
Why delete this method?

Reply:
We already have `ImageMode::num_chanels()`, trying to keep a single source of truth there.